package.json

```diff
@@ -108,7 +108,7 @@
   "husky": {
     "hooks": {
       "pre-commit": "lint-staged",
-      "pre-push": "yarn lint:js && yarn test:node"
+      "pre-push": "yarn lint:js && yarn test:ember"
```

Comment:
Would you mind also adding the node tests to this too?

Reply:
sure, done! 👍🏼